compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIESplitLogicalObjFifos.cpp

```diff
@@ -348,23 +348,87 @@ void AMDAIESplitLogicalObjFifosPass::runOnOperation() {
     return signalPassFailure();
   }
 
+  // Fetch and store all unique pairs of L2<->L1 Copy ops. This would helps us
+  // figure out the split factor for all LogicalObjectFifos.
+  DenseMap<Operation *, DenseSet<Operation *>> uniqueL2L1Pair;
```

Comment:
As you're just using the `.size()` of the `DenseSet`, you don't need to keep track of all ops, just how many there are.